client/app/services/style.service.js

```diff
@@ -43,14 +43,25 @@
         // The get style functions could check the cache for the style and if found return it, otherwise
         // create a new one and add to cache
         // var styleCache = {};
+        var d3Scale = null;
+
+        function setD3Scale(domain) {
+            if (!d3Scale) {
+                d3Scale = d3.scaleQuantile().domain(domain).range(d3.schemeReds[5]);
```

Comment:
We may want to bring this down to 4 or 3 eventually, but this is okay for now

Reply:
👍